src/transformers/modeling_marian.py

```diff
@@ -18,16 +18,26 @@
 from transformers.modeling_bart import BartForConditionalGeneration
 
 
-PRETRAINED_MODEL_ARCHIVE_MAP = {
-    "opus-mt-en-de": "https://cdn.huggingface.co/Helsinki-NLP/opus-mt-en-de/pytorch_model.bin",
-}
-
-
 class MarianMTModel(BartForConditionalGeneration):
     """Pytorch version of marian-nmt's transformer.h (c++). Designed for the OPUS-NMT translation checkpoints.
-    Model API is identical to BartForConditionalGeneration"""
-
-    pretrained_model_archive_map = PRETRAINED_MODEL_ARCHIVE_MAP
+    Model API is identical to BartForConditionalGeneration.
+    Available models are listed at https://huggingface.co/models?search=Helsinki-NLP
+
+    Examples:
+        src = 'fr'
+        trg = 'en'
```

Comment:
I don't think this will display the example correctly. I believe it should start with a blank line and have two semi-colons:
```suggestion
    Examples::
    
        src = 'fr'
        trg = 'en'
```